bazaar/src/lib.rs

```diff
@@ -98,11 +100,11 @@ decl_module! {
             ensure!(<encointer_communities::Module<T>>::community_identifiers().contains(&cid),
                 Error::<T>::InexistentCommunity);
 
-            ensure!(!BusinessRegistry::<T>::contains_key(cid, &sender), Error::<T>::ExistingBusiness);
+            let anonymous_proxy = pallet_proxy::Pallet::<T>::anonymous_account(&sender, &ProxyTypeOf::<T>::default(), 0, None);
```

Comment:
I am unsure of automatic proxy creation here.

Reasoning Against:
* We enforce the creation of a proxy account, which bloats runtime storage for businesses that don't need that.
* including the proxy pallet, couples the pallets.
* Permission configuration via the proxy filter happens on runtime level. Hence, it does not reduce the work that needs to be done there.

Reasoning Pro:
* It simplifies the client code because. Otherwise, we'd need to wrap a call to the proxy pallet
* if we want to make proxy, we need on extra extrinsic, but which business wants to have a proxy directly upon creation??

For me, the cons over weigh, what do you think @NicJak?

Reply:
I believe proxy creation should be standard. Busnisses may start without, but it is not possible to upgrade your business and make a reverse proxy out of an existing account (is it?)

however: "standard" could also be done by example by our client code. no need for the pallet to do that.

I'm in favor of the pallet not enforcing proxies but the client will default to creating one

Reply:
So, @NicJak may I ask you to remove the proxy creation code?